cloudpickle/cloudpickle.py

```diff
@@ -141,8 +143,22 @@ def _whichmodule(obj, name):
     return None
 
 
+if sys.version_info[:2] < (3, 7):
+    def _get_module_attr(obj):
+        if isinstance(obj, typing.TypeVar):
+            return None
+        return getattr(obj, '__module__', None)
+else:
+    def _get_module_attr(obj):
+        return getattr(obj, '__module__', None)
+
+
 def _is_global(obj, name=None):
```

Comment:
I think this is this PR is a good opportunity to rename this function to something more meaning full such as `_is_importable_by_name` or `_can_be_lookedup_by_name`.